proto/switch_state.proto

```diff
@@ -143,5 +143,8 @@ message SwitchState {
   message VLANState {
     // vlan acls states
     repeated ACLState acls = 1;
+
+    // packet rate state
+    State.State packet_rate_state = 2;
```

Comment:
Seems like this would be better aggregated by VLAN only, not by switch. Can you investigate how hard it would be to make this field a vlan map under dataplane_state?  dataplane_state.vlans[vlan].packet_rate_state

Reply:
Updated the structure